railties/test/application/configuration_test.rb

```diff
@@ -2472,12 +2472,12 @@ class ::DummySerializer < ActiveJob::Serializers::ObjectSerializer; end
       remove_from_config '.*config\.load_defaults.*\n'
 
       app_file "config/initializers/new_framework_defaults_6_0.rb", <<-RUBY
-        Rails.application.config.action_dispatch.return_only_media_type_on_content_type = true
+        Rails.application.config.action_dispatch.return_only_media_type_on_content_type = false
       RUBY
 
       app "development"
 
-      assert_equal true, ActionDispatch::Response.return_only_media_type_on_content_type
+      assert_equal false, ActionDispatch::Response.return_only_media_type_on_content_type
```

Comment:
I think you should use `assert_not` here instead of comparing booleans if that works for you.

```suggestion
      assert_not ActionDispatch::Response.return_only_media_type_on_content_type
```

Reply:
I just followed the convention in the existing line, but I guess the literal `false` (formerly `true`) is intentional, indicating that we're checking if the configuration value is properly set.

Reply:
In this case I think it's more important to be consistent with the two preceding tests:

https://github.com/rails/rails/blob/9a62e95f4a61ec010a146ade9c705909ed82c8ec/railties/test/application/configuration_test.rb#L2459

https://github.com/rails/rails/blob/9a62e95f4a61ec010a146ade9c705909ed82c8ec/railties/test/application/configuration_test.rb#L2468